apps/pie-docs/src/assets/styles/components/_homePage.scss

```diff
@@ -0,0 +1,110 @@
+@use '../fozzie-config' as f;
+
+.c-hero {
+    min-height: 100%;
+    display: grid;
+
+    // helps scale the razor with the page
+    grid-template-rows: minmax(0, 300px) minmax(0, 100px) auto;
+    background-color: f.$color-background-subtle;
+
+    @include f.media('>=mid') {
+        grid-template-rows: minmax(0, 200px) minmax(0, 200px) auto;
+    }
+
+    @include f.media('>=wide') {
+        grid-template-rows: minmax(0, 220px) minmax(0, 240px) auto;
+    }
+}
+
+.c-heroBanner,
```

Comment:
I think the format here should be `.c-hero-banner`. The dash is used to show hierarchy of the classes